Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -707,3 +707,381 @@
+    blacklist:
+      tag:
+      - Trash
+
+- type: pirateBounty
+  id: BountyPDA
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-pda
+    amount: 8
+    whitelist:
+      id:
+      - BasePDA
+
+- type: pirateBounty
+  id: BountyExtinguisher
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-extinguisher
+    amount: 5
+    whitelist:
+      id:
+      - FireExtinguisher
+
+- type: pirateBounty
+  id: BountyCaptainGloves
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-captainGloves
+    amount: 5
+    whitelist:
+      id:
+      - ClothingHandsGlovesCaptain
+
+- type: pirateBounty
+  id: BountyGyro
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-gyro
+    amount: 4
+    whitelist:
+      id:
+      - ShuttleGyroscope
+
+- type: pirateBounty
+  id: BountyDefib
+  reward: 3
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-defib
+    amount: 5
+    whitelist:
+      id:
+      - BaseDefibrillator
+
+- type: pirateBounty
+  id: BountyResearchDisk
+  reward: 5
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-researchDisk
+    amount: 7
+    whitelist:
+      id:
+      - ResearchDisk
+
+- type: pirateBounty
+  id: BountyAlcohol
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-alcohol
+    amount: 4
+    whitelist:
+      id:
+      - BoozeDispenser
+
+- type: pirateBounty
+  id: BountyThruster
+  reward: 4
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-thruster
+    amount: 4
+    whitelist:
+      id:
+      - BaseThruster
+
+- type: pirateBounty
+  id: BountyGravGen
+  reward: 2
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-gravGen
+    amount: 4
+    whitelist:
+      id:
+      - GravityGenerator
+
+- type: pirateBounty
+  id: BountyChemVend
+  reward: 3
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-chemVend
+    amount: 1
+    whitelist:
+      id:
+      - VendingMachineChemicals
+
+- type: pirateBounty
+  id: BountyVendingMachine
+  reward: 6
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-vendingmachine
+    amount: 3
+    whitelist:
+      id:
+      - VendingMachine
+
+- type: pirateBounty
+  id: BountyMercSuit
+  reward: 3
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-mercSuit
+    amount: 2
+    whitelist:
+      id:
+      - ClothingOuterHardsuitMercenary
+
+- type: pirateBounty
+  id: BountyCappy
+  reward: 1
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-cappy
+    amount: 10
+    whitelist:
+      id:
+      - MobCatCappy
+
+- type: pirateBounty
+  id: BountyCultistNecklace
+  reward: 1
+  description: bounty-description-pirate
+  entries:
+  - name: bounty-item-cultistNeck
+    amount: 4
```

Comment:
4 necklaces for 1 DB is too low, should be 1necklace for 4DB since at most an expedition crew could have 1.